ee/migrations/0009_migrate_definitions_tags.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.5 on 2022-01-28 19:21
+from django.db import migrations
+from django.db.models import Q
+
+from posthog.models.tagged_item import EnterpriseTaggedItem
+
+
+def forwards(apps, schema_editor):
+    # Create new event definition tags
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+    for instance in EnterpriseEventDefinition.objects.exclude(deprecated_tags__isnull=True, deprecated_tags=[]):
+        if instance.deprecated_tags:
+            for tag in instance.deprecated_tags:
+                new_tag = EnterpriseTaggedItem(
+                    event_definition_id=instance.eventdefinition_ptr_id, tag=tag, team_id=instance.team_id
+                )
+                new_tag.save()
```

Comment:
More of a note (because tags aren't really used today), but this could be a good opportunity to use django's `bulk_create`

https://docs.djangoproject.com/en/4.0/ref/models/querysets/#bulk-create

Reply:
nice!